### PR TITLE
DBZ-4947 Build Debezium UI upon changes in debezium-core

### PR DESCRIPTION
**Pull request by ani-sha**
https://issues.redhat.com/browse/DBZ-4947

### CODE REVIEW

**Comment by jpechane** [comment]
@ani-sha Can we have the UI build also part of the PR build as we have for Db2 and other non-core connectors?

**Comment by ani-sha** [comment]
@jpechane Yes. But do we want this kind of build also or just the PR build would be fine? This current workflow is triggered only upon `push` in the UI repository.

**Comment by jpechane** [comment]
@ani-sha Ideally I'd like to have a PR failure if Debezium UI build is not working with the new PR.

**Comment by jpechane** [comment]
@ani-sha Could you please take a look? The UI build is failing.

File: .github/workflows/debezium-workflow.yml [hunk 1]
```diff
@@ -768,3 +781,36 @@ jobs:
...
+          key: maven-debezium-test-build-${{ hashFiles('core/**/pom.xml') }}
+          restore-keys: |
+            maven-debezium-test-build-${{ hashFiles('core/**/pom.xml') }}
+
+      - name: Build Debezium UI
+        run: >
+          ./mvnw clean install -f debezium-ui/backend/pom.xml -am -pl backend -B
+          -Ddebezium.test.records.waittime=20
+          -Dformat.formatter.goal=validate
+          -Dformat.imports.goal=check -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
```
Comment (ani-sha) [inline .github/workflows/debezium-workflow.yml:816]: @jpechane I tried running with this command, but it fails. What do you think would be the right command?

Comment (Naros) [reply .github/workflows/debezium-workflow.yml:816]: Hi @ani-sha so the issue is that the UI uses the Maven wrapper which is only available from the `./debezium-ui` path so perhaps for this build step we should use:
```
cd debezium-ui && ./mvnw clean install -f backend/pom.xml ...
```
Give that a try and see if that works.

Comment (Naros) [reply .github/workflows/debezium-workflow.yml:816]: Another option is to add `working-directory` to the build step and specify `debezium-ui` and omit the `cd` portion of the runnable command too.

Comment (jpechane) [reply .github/workflows/debezium-workflow.yml:816]: Already tested in https://github.com/debezium/debezium/pull/3481
I am going to get the commit here
But we need to fix the UI build first
It fails for me locally too

**Comment by jpechane** [comment]
The merge is blocked by https://issues.redhat.com/browse/DBZ-5125

**Comment by jpechane** [comment]
@ani-sha Applied, thanks a lot. The UI tests are now passing.

**Pull request merged**